eslint-config-base/index.js

```diff
@@ -190,5 +388,33 @@ module.exports = {
     'import/no-self-import': 1,
     'import/no-cycle': 0,
     'import/no-relative-parent-imports': 0,
+    'import/prefer-default-export': 0,
+    'import/no-extraneous-dependencies': [2, {
+      devDependencies: [
+        'test/**', // tape, common npm pattern
+        'tests/**', // also common npm pattern
+        'spec/**', // mocha, rspec-like pattern
+        '**/__tests__/**', // jest pattern
+        '**/__mocks__/**', // jest pattern
+        'test.{js,jsx}', // repos with a single test file
+        'test-*.{js,jsx}', // repos with multiple top-level test files
+        '**/*{.,_}{test,spec}.{js,jsx}', // tests where the extension or filename suffix denotes that it is a test
+        '**/jest.config.js', // jest config
+        '**/jest.setup.js', // jest setup
+        '**/vue.config.js', // vue-cli config
+        '**/webpack.config.js', // webpack config
+        '**/webpack.config.*.js', // webpack config
+        '**/rollup.config.js', // rollup config
+        '**/rollup.config.*.js', // rollup config
+        '**/gulpfile.js', // gulp config
+        '**/gulpfile.*.js', // gulp config
+        '**/Gruntfile{,.js}', // grunt config
+        '**/protractor.conf.js', // protractor config
+        '**/protractor.conf.*.js', // protractor config
+        '**/karma.conf.js', // karma config
+        '**/.eslintrc.js' // eslint config
```

Comment:
looks like a lot of items from tools we never used or are on Hold in out Tech Radar